src/sagas/wallet.js

```diff
@@ -117,7 +117,6 @@ export function* startWallet(action) {
     words,
     pin,
   } = action.payload;
-  NavigationService.navigate('LoadHistoryScreen');
```

Comment:
This is not a screen that belongs to any navigator, but rather an overlay `View` over the `AppStack` navigator. It cannot be navigated to.